blosc/blosc.c

```diff
@@ -59,7 +59,6 @@
 /* Include the win32/pthread.h library for all the Windows builds. See #224. */
```

Comment:
The comment explains why the .c file is included, please remove the comment too.

Reply:
done